include/libdnf/conf/option_bool.hpp

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (C) 2018-2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_CONF_OPTION_BOOL_HPP
+#define LIBDNF_CONF_OPTION_BOOL_HPP
+
+
+#include "option.hpp"
+
+#include <array>
+#include <memory>
+#include <vector>
+
+namespace libdnf {
+
+
+class OptionBool : public Option {
+public:
+    using ValueType = bool;
+
+    class InvalidValue : public Option::InvalidValue {
+    public:
+        using Option::InvalidValue::InvalidValue;
+        const char * get_domain_name() const noexcept override { return "libdnf::OptionBool"; }
+    };
+
+    explicit OptionBool(bool default_value);
+    OptionBool(
+        bool default_value, const std::vector<std::string> & true_vals, const std::vector<std::string> & false_vals);
+    OptionBool(const OptionBool & src);
+    OptionBool(OptionBool && src) noexcept = default;
+    ~OptionBool() override = default;
+
+    OptionBool & operator=(const OptionBool & src);
+    OptionBool & operator=(OptionBool && src) noexcept = default;
+
+    OptionBool * clone() const override;
+    void test(bool /*unused*/) const;
+    bool from_string(const std::string & value) const;
```

Comment:
Please could you add a description? It is Important to know that it only converts value from string to bool. Should we expose it as an API?